Dockerfile

```diff
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-FROM golang:1.9.2
+FROM golang:1.12.0
```

Comment:
I believe this tag should be `1.12` or specify a newer minor version tag like `1.12.12`. Personally I think it would be best to use `1.12.12` here.

Reply:
Here is the tag info for 1.12.0 from DockerHub ... https://hub.docker.com/_/golang?tab=tags&page=1&name=1.12.0

Reply:
Sounds good to me, updated

Reply:
LGTM